src/collection-view.js

```diff
@@ -482,7 +487,13 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // render the child view
   renderChildView: function(view, index) {
+    if (!(view instanceof Marionette.View)) {
+      Marionette.triggerMethodOn(view, 'before:render', view);
+    }
     view.render();
+    if (!(view instanceof Marionette.View)) {
```

Comment:
You mean the order of fire `before:render` -> call `render()` -> fire `render`?  I am trying to maintain the natural order, such as in [ItemView#render](https://github.com/marionettejs/backbone.marionette/blob/4c00dabc3f41a95edac2773824ebecaf6c130986/src/item-view.js#L54-L60).